includes/core/classes/class-assets.php

```diff
@@ -176,7 +176,8 @@ function ( $key ) {
 			array_keys( $settings->get_sub_pages() )
 		);
 
-		if ( in_array( $hook, $setting_hooks, true ) ) {
+		// Had to add the profile page to the check since it uses some of the same assets.
+		if ( in_array( $hook, $setting_hooks, true ) || 'profile.php' === $hook ) {
```

Comment:
Let's not put in settings as that's the wrong place. I would put in it's own JS file or in admin.js for now.

Reply:
Not sure what you mean by this.

Reply:
Refactored both this and the settings one to use a common helper.